packages/shell-api/src/helpers.ts

```diff
@@ -403,8 +403,8 @@ export async function getPrintableShardStatus(db: Database, verbose: boolean): P
 
   // Special case the config db, since it doesn't have a record in config.databases.
   databases.push({ '_id': 'config', 'primary': 'config', 'partitioned': true });
-  databases.sort((a: any, b: any): any => {
-    return a._id > b._id;
+  databases.sort((a: any, b: any): number => {
+    return a._id.localeCompare(b._id);
```

Comment:
Just something I noticed, this would obviously not have worked before because the `.sort()` callback should not return a boolean.